index.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     let project = this.project
     let ui = this.ui
 
-    if (type === 'templates' || type === 'addon') {
```

Comment:
unless something has changed with will break linting in NON addons it will make it lint all the addons used by a project  which obviously do not follow our lint rules

Reply:
Something must have changed.  It doesn't seem to be doing it in non-addons.  I just checked to make sure.  `ember-cli-eslint` seems to just restrict on `templates` as well: https://github.com/ember-cli/ember-cli-eslint/blob/master/index.js#L38